docs/new_problem.md

```diff
@@ -240,16 +240,40 @@ All hyperparamters inherit from `_default_hparams()` in `problem.py.` If you wou
 from tensor2tensor.models import transformer
 
 @registry.register_hparams
-def word2def_hparams(self):
+def word2def_hparams():
     hparams = transformer.transformer_base_single_gpu()  # Or whatever you'd like to build off.
     hparams.batch_size = 1024
     return hparams
 ```
 
+# Test the data generation
+
+You can test data generation of your a problem in your own project with:
+
+```bash
+PROBLEM=word2def
+DATA_DIR=$HOME/t2t_data
+TMP_DIR=/tmp/t2t_datagen
+mkdir -p $DATA_DIR $TMP_DIR
+
+t2t-datagen \
+  --t2t_usr_dir=$PATH_TO_YOUR_PROBLEM_DIR \
+  --data_dir=$DATA_DIR \
+  --tmp_dir=$TMP_DIR \
+  --problem=$PROBLEM
+```
+
+Where:
+*   `PROBLEM` is the name of the class that was registered with `@registry.register_problem()`, but converted from `CamelCase` to `snake_case`.
+*   `PATH_TO_YOUR_PROBLEM_DIR` is a path to the directory of your python problem file.
+
+If you are adding to the tensor2tensor repository, you can install the local cloned version in developer mode with `pip installe -e .` from the tensor2tensor directory. You will also need to add your new problem file to [`all_problems.py`](https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/data_generators/all_problems.py).
```

Comment:
Great, now just `installe` -> `install` and maybe also
are adding -> plan to contribute
will also need -> can also
(because not all contributions involve adding a new problem definition)

Reply:
cool, done.